apps/passport-server/src/services/issuanceService.ts

```diff
@@ -98,6 +103,7 @@ export const ZUZALU_23_VISITOR_PRODUCT_ID =
 export const ZUZALU_23_ORGANIZER_PRODUCT_ID =
   "10016d35-40df-4033-a171-7d661ebaccaa";
 export const ZUZALU_23_EVENT_ID = "5de90d09-22db-40ca-b3ae-d934573def8b";
+export const ZUCONNECT_23_EVENT_ID = "91312aa1-5f74-4264-bdeb-f4a3ddb8670c";
```

Comment:
could it be worth defining the zuconnect product ids here too, or at least commenting and pointing the reader in the direction of where they are defined?